release/src/router/rom/webs_scripts/merlin_webs_update.sh

```diff
@@ -2,7 +2,7 @@
 set -u;
 wget_options="-q -t 2 -T 30"
 
-fwsite="https://raw.githubusercontent.com/gnuton/asuswrt-merlin.ng/master/updates/"
+fwsite="https://raw.githubusercontent.com/gnuton/asuswrt-merlin.ng/master/updates"
```

Comment:
the additional / is not required since the "fwsite" variable has a "/" infront already.

Reply:
> the additional / is not required since the "fwsite" variable has a "/" infront already.
>
Yes, good catch. But the reasoning is that when referencing the "**fwsite**" variable, the code already has the forward slash char in front of it so the variable itself does not need to include the slash.